commit 78321aeae5002be7e54b7258cc5f6c605ed4e147
Author: Dan Carpenter <dan.carpenter@oracle.com>
Date:   Mon Dec 17 10:08:54 2018 +0300

    staging: bcm2835-audio: double free in init error path
    
    commit 136ff5e49271c4c8fceeca5491c48e66b961564b upstream.
    
    We free instance here and in the caller.  It should be only the caller
    which handles it.
    
    Fixes: d7ca3a71545b ("staging: bcm2835-audio: Operate non-atomic PCM ops")
    Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
    Reviewed-by: Takashi Iwai <tiwai@suse.de>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-vchiq.c b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-vchiq.c
index cf5ae1a28f52..ea78e6aade7f 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-vchiq.c
+++ b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-vchiq.c
@@ -145,7 +145,6 @@ vc_vchi_audio_init(VCHI_INSTANCE_T vchi_instance,
 		dev_err(instance->dev,
 			"failed to open VCHI service connection (status=%d)\n",
 			status);
-		kfree(instance);
 		return -EPERM;
 	}
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1481_78321aeae5002be7e54b7258cc5f6c605ed4e147.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1481_78321aeae5002be7e54b7258cc5f6c605ed4e147.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
